### ISSUE TITLE
Add option for 24hr time for time facet via settings

### ISSUE THREAD

**Issue by 989onan** [Yellow-Dog-Man/Resonite-Issues#1277]
### Is your feature request related to a problem? Please describe.

As said in #614 , 
 I use 24hr as an American to try to match the rest of the world. I don't see any way of changing the time facet from 12hr to 24hr without using ui edit mode and editing the facet.

### Describe the solution you'd like

What would be nice is to change the default time facet in dash to allow people to choose 24hr vs 12hr time. This would be preferable to do through the language menu in the upcoming settings ui from #614.

### Describe alternatives you've considered

I had to edit the facet to put it in 24 hour time, which is thankfully an enum or checkbox inside of it if I remember correctly. Editing it via ui edit mode is okay for an intermediate user, but is a tall task for a new user

**Comment by 989onan** [comment]
Also accidentally opened this as an enhancement because it felt better in that category based on the form process and descriptions in said process

**Comment by Nytra** [comment]
It's actually strange because for me the Time facet does use a 24hr format by default. This is because of the way C# formats time based on the current culture. There is a difference between which geographical location your system is set to.

**Comment by Banane9** [comment]
Yup, getting 24h time on de-DE too. Ultimately it would be nice, if all formatting would respect the locale settings and could be overwritten as well, e.g. using US-english text and 24h time format.

**Comment by Frooxius** [comment]
Facets are technically content, so they're typically not configured via general settings.

If you want this specifically for the facet,we'd probably have that done on the facet, rather than settings.

What would make more sense for settings is a global default date/time locale.

**Comment by AegisTheWolf** [comment]
This issue is related to this other issue with the same underlying problem (missing time locale) https://github.com/Yellow-Dog-Man/Resonite-Issues/issues/185
